src/bit-systems/delete-entity-system.ts

```diff
@@ -0,0 +1,49 @@
+import { defineQuery, exitQuery, hasComponent, removeEntity } from "bitecs";
+import { Vector3 } from "three";
+import { HubsWorld } from "../app";
+import { Deletable, HoveredRemoteLeft, HoveredRemoteRight } from "../bit-components";
+import { paths } from "../systems/userinput/paths";
+import { animate } from "../utils/animate";
+import { findAncestorEntity } from "../utils/bit-utils";
+import { coroutine } from "../utils/coroutine";
+import { easeOutQuadratic } from "../utils/easing";
+
+// TODO Move to coroutine.ts when it exists
+// TODO Figure out the appropriate type and use it everywhere
+export type Coroutine = Generator<Promise<void>, void, unknown>;
+
+const END_SCALE = new Vector3().setScalar(0.001);
+function* deleteEntity(world: HubsWorld, eid: number): Coroutine {
```

Comment:
We mix "delete" and "remove" here. I thought we should maybe use `remove` as that's what both BitECS and THREE use. This function would then clash but it can be called `animatedRemoveEntity`, `animateThenRemove` or `animateRemoval`. I do wonder if there is value in differentiating though, because lots of things can "remove" entities. But maybe the user is "deleting" them? This issue became more apparent when I tried to think of the name for the component to tag these.